run_unit_tests.sh

```diff
@@ -31,6 +32,9 @@ ${debug_bazel_flags} \
 --test_summary=detailed \
 --test_output=streamed"
 
+LD_LIBRARY_PATH=/opt/opencv/lib/:/opt/intel/openvino/runtime/lib/intel64/:/opt/intel/openvino/runtime/3rdparty/tbb/lib/
+PYTHONPATH=/opt/intel/openvino/python:/ovms/bazel-bin/src/python/binding
```

Comment:
Can we add these values to existing `PYTHONPATH` ?
Just in case this would be run for bare metal deployment after setupvars has been called.

Reply:
@mzegla it was copied from bazelrc. less content should ensure we don't have conflicts. This is temporary till python3.12 interpreter is fixed...